tests/js/spec/components/avatar/avatarList.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import AvatarList from 'app/components/avatar/avatarList';
+
+function renderComponent(avatarUsersSixUsers: AvatarList['props']['users']) {
+  return mountWithTheme(<AvatarList users={avatarUsersSixUsers} />);
+}
+
+describe('AvatarList', () => {
+  it('renders with user letter avatars', () => {
+    const users = [
+      // @ts-expect-error
+      TestStubs.User({id: '1', name: 'AB'}),
+      // @ts-expect-error
+      TestStubs.User({id: '2', name: 'BC'}),
+    ];
+
+    const {container, queryByTestId, getByText} = renderComponent(users);
+    expect(getByText('A')).toBeTruthy();
+    expect(getByText('B')).toBeTruthy();
+    expect(queryByTestId('avatarList-collapsedusers')).toBeNull();
+
+    expect({html: () => container.innerHTML}).toSnapshot();
```

Comment:
This was needed because our visual snapshot package currently only accepts the `html()` enzyme function.

https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/html.html

https://github.com/getsentry/jest-visual-snapshot/blob/main/packages/jest/toSnapshot.js

Reply:
I've updated the package to support this (although I haven't tested it) https://github.com/getsentry/jest-visual-snapshot/blob/main/packages/jest/toSnapshot.js#L6

Reply:
Nice! Thank you 🙏

Reply:
it works 😉